.github/workflows/housekeeping_aks_issuejson.yaml

```
@@ -27,7 +27,7 @@ jobs:
       - name: Delete old entries
         continue-on-error: true
         run: |
-          python housekeeping.py
+          python ./scripts/aks/housekeeping.py
```

I don't think this script is aks specific?

```suggestion
          python ./scripts/housekeeping.py
```